financial-templates-lib/price-feed/CreatePriceFeed.js

```diff
@@ -186,12 +186,17 @@ async function createUniswapPriceFeedForEmp(logger, web3, networker, getTime, em
  * @return {Object} an instance of PriceFeedInterface that can be used to get the reference price.
  */
 async function createReferencePriceFeedForEmp(logger, web3, networker, getTime, empAddress, config) {
+  const emp = getEmpAtAddress(web3, empAddress);
+
+  // Infer lookback from liquidation liveness.
+  const lookback = Number((await emp.methods.liquidationLiveness().call()).toString());
+
   // TODO: maybe move this default config to a better location.
   const defaultConfigs = {
     "ETH/BTC": {
       type: "medianizer",
       pair: "ethbtc",
-      lookback: 7200,
+      lookback,
       minTimeBetweenUpdates: 60,
```

Comment:
I see you passed `minTimeBetweenUpdates` in the `config` object param, why not read it dynamically from that?

Reply:
perhaps because this is a default?

Reply:
Yeah, exactly. These are the default values. In the test, I override the minTimeBetweenUpdates to test the override functionality.

Reply:
@nicholaspai lmk if this looks good to you.

Reply:
looks good